rpc/jsonrpc/types/chainsvrresults.go

```diff
@@ -28,6 +28,11 @@ type DecodeScriptResult struct {
 
 // EstimateSmartFeeResult models the data returned from the estimatesmartfee
 // command.
+//
+// NOTE: EstimateSmartFee currently returns a float. This result type is
+// unused.
+//
+// TODO: Rework EstimateSmartFee to return this result type.
```

Comment:
No longer deprecated.

Reply:
Will also create an issue unless you would like to.

Reply:
Thanks for update.  I'll let you open the issue since you essentially spotted the discrepancy.